cmd/exrun

```diff
@@ -63,7 +62,7 @@ if [ "$1" == "-h" ]; then
     skip_autostart=y
 fi
 
-export PYTHONPATH=$FORCH:$FAUCET:$MININET:$LIBS
+export PYTHONPATH=$FORCH:$FAUCET:$MININET:$ROOT
```

Comment:
Is this ROOT necessary?

Reply:
Yes -- took me a while to figure it all out -- but the grpc generator uses absolute package names, so it's daq.proto.XXX. So, the $ROOT is necessary to find the import by absolute package name, since the basic "python daq/entry.py" runs in the subdirectory as the (nameless package root) and so doesn't no how to find an import daq.proto.XXX (it would require a relative import).  It's all just a rats-nest of dependencies!